src/components/Home.js

```diff
@@ -117,23 +85,22 @@ export class Home extends React.Component {
               : (
                 <div>
                   <Toolbar
-                    diffArray={this.state.diffArray} onMenuClick={this.handleMenuClick} onLowercase={this.handleLowercase}
+                    diffArray={this.props.diffArray} onMenuClick={this.handleMenuClick} onLowercase={this.handleLowercase}
```

Comment:
* `Toolbar` should not depend on props which can be taken from `store`
* it makes sense to incapsulate event handlers into the `Toolbar`

Reply:
Same for elements below